tasks/register_client.yml

```diff
@@ -1,15 +1,11 @@
 ---
-- name: Register client {{ item.name }}
-  shell: >
-    echo
-    "configure add client "
```

Comment:
From a previous discussion, I thought the command `configure add client ...` was adding the client to the DB.

If my understanding is correct here, a new client will be added to the database once the bareos server will be reloaded, correct?

Reply:
I guess (I haven't been able to confirm reading it anywhere)
We're simply following https://docs.bareos.org/IntroductionAndTutorial/Tutorial.html#manual-configuration-of-client : 2 methods can be used : either the `configure add` command, or creating the conf files and reloading.

Reply:
Nice! I definitely prefer your new proposal.